README.md

```diff
@@ -1,8 +1,8 @@
 <div align="center">
-<img src="./assets/logo-burn-full.png" width="200px"/>
+<img src="https://raw.githubusercontent.com/burn-rs/burn/main/assets/logo-burn-full.png" width="200px"/>
 
 [![Current Crates.io Version](https://img.shields.io/crates/v/burn.svg)](https://crates.io/crates/burn)
-[![Test Status](https://github.com/burn-rs/burn/actions/workflows/test.yml/badge.svg)](https://github.com/burn-rs/burn/actions/workflows/test.yml)
+[![Test Status](https://img.shields.io/github/workflow/status/burn-rs/burn/test?label=test&logo=github)](https://github.com/burn-rs/burn/actions/workflows/test.yml)
```

Comment:
The test status on the `main` branch is already fixed, but not actually deployed on crate.io, so I would remove the change here, but thanks for the image :)